brainvisa/toolboxes/spm/processes/spm12/tools/shoot_tools/SPM12ShootWriteNormalised_generic.py

```diff
@@ -0,0 +1,111 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
```

Comment:
Good to know, those lines are not necessary now. 
Since python3, the default encoding is utf-8, so the `coding` line is not mandatory now.
`__future__` import is intended to be used by files supporting python2 and python3, but as there is no absolute import in your file, this import is not necessary.

Reply:
Thanks for the info, I will remove that. I copy/past python scripts first lines a little bit to automatically ;).